.circleci/config.yml

```diff
@@ -9,7 +9,7 @@ orbs:
 references:
   container_config: &container_config
     docker:
-      - image: circleci/node
+      - image: circleci/node:10
```

Comment:
I'm curious, is there any value in specifying the full semver version we want?
https://hub.docker.com/r/circleci/node/tags?page=1&ordering=-name

Reply:
I don't think so. It also looks like the other versions have not been maintained for the last 2 years. My assumption is that `10` is the best practice.

Atlantis works (afaik) in all node 10 versions.